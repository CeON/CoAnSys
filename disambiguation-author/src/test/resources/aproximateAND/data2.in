a1	1	Kowalski	[EX_TITLE#{(An Empirical Study on the AND)},EX_YEAR#{(2011)}]
a2	2	Kowalski	[EX_TITLE#{(An Empirical Study on the AND)},EX_YEAR#{(2012)}]
a3	3	Kowalski	[EX_TITLE#{(An Empirical Study on the BAND)},EX_YEAR#{(2013)}]
a4	3	Kowalski	[EX_TITLE#{(An Empirical Study on the BAND)},EX_YEAR#{(2014)}]
a5	2	Kowalski	[EX_TITLE#{(An Empirical Study on the AND)},EX_YEAR#{(2014)}]
b1	2	Kowalski	[EX_TITLE#{(An Empirical Study on the TRAND)},EX_YEAR#{(1980)}]
b2	2	Kowalski	[EX_TITLE#{(An Empirical Study on the BRAND)},EX_YEAR#{(1980)}]
b3	2	Kowalski	[EX_TITLE#{(An Empirical Study on the SAND)},EX_YEAR#{(1990)}]
b4	2	Kowalski	[EX_TITLE#{(An Empirical Study on the BRAND)},EX_YEAR#{(1990)}]
c1	1	Nowak	[EX_TITLE#{(An Empirical Study on the AND)},EX_YEAR#{(2011)}]
c2	2	Nowak	[EX_TITLE#{(An Empirical Study on the AND)},EX_YEAR#{(2012)}]
c3	3	Nowak	[EX_TITLE#{(An Empirical Study on the BAND)},EX_YEAR#{(2013)}]
c4	3	Nowak	[EX_TITLE#{(An Empirical Study on the BAND)},EX_YEAR#{(2014)}]
c5	2	Nowak	[EX_TITLE#{(An Empirical Study on the AND)},EX_YEAR#{(2014)}]
c6	2	Nowak	[EX_TITLE#{(An Empirical Study on the TRAND)},EX_YEAR#{(1980)}]
c7	2	Nowak	[EX_TITLE#{(An Empirical Study on the BRAND)},EX_YEAR#{(1980)}]
c8	2	Nowak	[EX_TITLE#{(An Empirical Study on the SAND)},EX_YEAR#{(1990)}]
c9	2	Nowak	[EX_TITLE#{(An Empirical Study on the BRAND)},EX_YEAR#{(1990)}]
c10	1	Nowak	[EX_TITLE#{(An Empirical Study on the AND)},EX_YEAR#{(2000)}]
c11	2	Nowak	[EX_TITLE#{(An Empirical Study on the TRAND)},EX_YEAR#{(2000)}]
d1	1	Wos	[EX_TITLE#{(Tit)},EX_YEAR#{(2000)}]
d2	1	Wos	[EX_TITLE#{(Tit)},EX_YEAR#{(2000)}]
d3	1	Wos	[EX_TITLE#{(Tit)},EX_YEAR#{(2000)}]
d4	1	Wos	[EX_TITLE#{(Tit)},EX_YEAR#{(2000)}]
d5	1	Wos	[EX_TITLE#{(Tit)},EX_YEAR#{(2000)}]
d6	1	Wos	[EX_TITLE#{(Tit)},EX_YEAR#{(2000)}]
d7	1	Wos	[EX_TITLE#{(Tit)},EX_YEAR#{(2000)}]
d8	1	Wos	[EX_TITLE#{(Tit)},EX_YEAR#{(2000)}]
d9	1	Wos	[EX_TITLE#{(Tit)},EX_YEAR#{(2000)}]
d10	1	Wos	[EX_TITLE#{(Tit)},EX_YEAR#{(2000)}]
d11	1	Wos	[EX_TITLE#{(Tit)},EX_YEAR#{(2000)}]
d12	1	Wos	[EX_TITLE#{(Tit)},EX_YEAR#{(2000)}]
d13	1	Wos	[EX_TITLE#{(Tit)},EX_YEAR#{(2000)}]
d14	1	Wos	[EX_TITLE#{(Tit)},EX_YEAR#{(2000)}]
d15	1	Wos	[EX_TITLE#{(Tit)},EX_YEAR#{(2000)}]
