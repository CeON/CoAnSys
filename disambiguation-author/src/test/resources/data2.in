c1	1	Kowalski	[EX_TITLE#{(An Empirical Study on the AND)},EX_YEAR#{(2011)}]
c2	2	Kowalski	[EX_TITLE#{(An Empirical Study on the AND)},EX_YEAR#{(2011)}]
c3	3	Kowalski	[EX_TITLE#{(An Empirical Study on the BAND)},EX_YEAR#{(2011)}]
c4	1	Nowak	[EX_TITLE#{(Kosmiczny Calkiem Dlugi Tytul)},EX_YEAR#{(1996)}]
c5	1	Nowak	[EX_TITLE#{(Kosmiczny Calkiem Dlugi Tytul 2)},EX_YEAR#{(1996)}]
c6	1	Nowak	[EX_TITLE#{(Kosmiczny Calkiem Dlugi Tytul 3)},EX_YEAR#{(1996)}]
c7	1	Nowak	[EX_TITLE#{(Kosmiczny Calkiem Dlugi Tytul 4)},EX_YEAR#{(1996)}]
c8	1	Nowak	[EX_TITLE#{(Kosmiczny Tytul)},EX_YEAR#{(1996)}]
c9	1	Nowak	[EX_TITLE#{(Kosmiczny Tytul)},EX_YEAR#{(1996)}]
c10	1	Nowak	[EX_TITLE#{(Kosmiczny Tytul)},EX_YEAR#{(1996)}]
c11	1	Nowak	[EX_TITLE#{(Kosmiczny Calkiem Tytul 2)},EX_YEAR#{(1996)}]
c12	1	Nowak	[EX_TITLE#{(Samotnik)},EX_YEAR#{(2001)}]
