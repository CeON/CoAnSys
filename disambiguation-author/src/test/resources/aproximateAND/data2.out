({(d8,Wos,[EX_YEAR#{(2000)},EX_TITLE#{(Tit)}]),(d7,Wos,[EX_YEAR#{(2000)},EX_TITLE#{(Tit)}]),(d15,Wos,[EX_YEAR#{(2000)},EX_TITLE#{(Tit)}]),(d14,Wos,[EX_YEAR#{(2000)},EX_TITLE#{(Tit)}]),(d13,Wos,[EX_YEAR#{(2000)},EX_TITLE#{(Tit)}]),(d1,Wos,[EX_YEAR#{(2000)},EX_TITLE#{(Tit)}]),(d2,Wos,[EX_YEAR#{(2000)},EX_TITLE#{(Tit)}]),(d3,Wos,[EX_YEAR#{(2000)},EX_TITLE#{(Tit)}]),(d4,Wos,[EX_YEAR#{(2000)},EX_TITLE#{(Tit)}]),(d5,Wos,[EX_YEAR#{(2000)},EX_TITLE#{(Tit)}]),(d6,Wos,[EX_YEAR#{(2000)},EX_TITLE#{(Tit)}]),(d12,Wos,[EX_YEAR#{(2000)},EX_TITLE#{(Tit)}]),(d11,Wos,[EX_YEAR#{(2000)},EX_TITLE#{(Tit)}]),(d10,Wos,[EX_YEAR#{(2000)},EX_TITLE#{(Tit)}]),(d9,Wos,[EX_YEAR#{(2000)},EX_TITLE#{(Tit)}])},{(1,0,1.0),(2,0,1.0),(3,0,1.0),(4,0,1.0),(5,0,1.0),(6,0,1.0),(7,0,1.0),(8,0,1.0),(9,0,1.0),(10,0,1.0),(11,0,1.0),(12,0,1.0),(13,0,1.0),(14,0,1.0)})
({(c9,Nowak,[EX_YEAR#{(1990)},EX_TITLE#{(An Empirical Study on the BRAND)}]),(c1,Nowak,[EX_YEAR#{(2011)},EX_TITLE#{(An Empirical Study on the AND)}]),(c2,Nowak,[EX_YEAR#{(2012)},EX_TITLE#{(An Empirical Study on the AND)}]),(c3,Nowak,[EX_YEAR#{(2013)},EX_TITLE#{(An Empirical Study on the BAND)}]),(c4,Nowak,[EX_YEAR#{(2014)},EX_TITLE#{(An Empirical Study on the BAND)}]),(c5,Nowak,[EX_YEAR#{(2014)},EX_TITLE#{(An Empirical Study on the AND)}]),(c6,Nowak,[EX_YEAR#{(1980)},EX_TITLE#{(An Empirical Study on the TRAND)}]),(c7,Nowak,[EX_YEAR#{(1980)},EX_TITLE#{(An Empirical Study on the BRAND)}]),(c8,Nowak,[EX_YEAR#{(1990)},EX_TITLE#{(An Empirical Study on the SAND)}]),(c10,Nowak,[EX_YEAR#{(2000)},EX_TITLE#{(An Empirical Study on the AND)}]),(c11,Nowak,[EX_YEAR#{(2000)},EX_TITLE#{(An Empirical Study on the TRAND)}])},{(2,1,0.0),(4,3,0.0),(5,1,0.0),(5,4,0.0),(7,0,0.0),(7,6,0.0),(8,0,0.0),(9,1,0.0),(10,6,0.0),(10,9,0.0)})
({(b4,Kowalski,[EX_YEAR#{(1990)},EX_TITLE#{(An Empirical Study on the BRAND)}]),(b3,Kowalski,[EX_YEAR#{(1990)},EX_TITLE#{(An Empirical Study on the SAND)}]),(b2,Kowalski,[EX_YEAR#{(1980)},EX_TITLE#{(An Empirical Study on the BRAND)}]),(b1,Kowalski,[EX_YEAR#{(1980)},EX_TITLE#{(An Empirical Study on the TRAND)}])},{(1,0,0.0),(2,0,0.0),(3,2,0.0)})
({(a5,Kowalski,[EX_YEAR#{(2014)},EX_TITLE#{(An Empirical Study on the AND)}]),(a4,Kowalski,[EX_YEAR#{(2014)},EX_TITLE#{(An Empirical Study on the BAND)}]),(a3,Kowalski,[EX_YEAR#{(2013)},EX_TITLE#{(An Empirical Study on the BAND)}]),(a2,Kowalski,[EX_YEAR#{(2012)},EX_TITLE#{(An Empirical Study on the AND)}]),(a1,Kowalski,[EX_YEAR#{(2011)},EX_TITLE#{(An Empirical Study on the AND)}])},{(1,0,0.0),(2,1,0.0),(3,0,0.0),(4,0,0.0)})
